pkg/models/tagged_html.go

```diff
@@ -0,0 +1,216 @@
+package models
+
+import (
+	"context"
+	"database/sql/driver"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"regexp"
+
+	"github.com/google/uuid"
+	"github.com/samber/lo"
+
+	"github.com/cms-enterprise/easi-app/pkg/appcontext"
+	"github.com/cms-enterprise/easi-app/pkg/sanitization"
+)
+
+var (
+	spanRe      = regexp.MustCompile(`<span[^>]*>.*?</span>`)
+	attributeRe = regexp.MustCompile(`(\S+)="([^"]+)"`)
+)
+
+// TaggedHTML Is the input type for HTML that could contain tags
+type TaggedHTML TaggedContent
+
+// TaggedContent represents rich text HTML with possible tagged HTML
+type TaggedContent struct {
+	RawContent HTML
+	Tags       []*Tag
+}
```

Comment:
Do we still want/need this type?

Reply:
Or should we just rename it to `TaggedHTML` and just have the 1 struct definition?